expertise/service/utils.py

```diff
@@ -692,31 +692,53 @@ def set_client(self, client):
         self.client = client
 
     def _generate_vertex_prefix(api_request):
```

Comment:
This function is doing two things:
- Validating the api request
- Returning the vertex prefix

I think the validation of the api request should be done way before we get to this point and in a separate function.
That way when we get to this function you can simply create the vertex prefix from the request without worrying about validation

Reply:
That's true, it makes the purpose of the function confusing, the validation does happen earlier before getting to the `create_job` function in the worker thread:
https://github.com/openreview/openreview-expertise/blob/ddcd5fb0dd2a2a05216b4af83847945ed893e1cf/expertise/service/expertise.py#L902
and the request gets validated through these two functions:
https://github.com/openreview/openreview-expertise/blob/ddcd5fb0dd2a2a05216b4af83847945ed893e1cf/expertise/service/expertise.py#L349-L359